app/lib/site_wide_search.rb

```diff
@@ -101,8 +108,4 @@ def tags_filter
   def highlight
     index.highlight(max_analyzed_offset: 999_000, fields: { fulltext: {}, deftext: {} })
   end
-
```

Comment:
This code was add probably to resolve items ordering issue, but after we implemented script-based ordering it become obsolete, so I've removed it